### PR TITLE
feat: Keygen migration script

### PR DESCRIPTION
**Pull request by kingpinXD**
# Description

Add a migration script to set status to `KeyGenSuccess` if its in Pending Status

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

File: app/setup_handlers.go [hunk 1]
```diff
@@ -34,3 +37,12 @@ func SetupHandlers(app *App) {
 		app.SetStoreLoader(types.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
 }
+
+type VersionMigrator struct {
+	v module.VersionMap
+}
+
+func (v VersionMigrator) TriggerMigration(moduleName string) module.VersionMap {
```
Comment (lumtis) [inline app/setup_handlers.go:45]: Why is it needed if we increase `ConsensusVersion`?

**Review by lumtis — COMMENTED** [review]
Has it been tested with `make stateful-upgrade-source`?

**Review by lumtis — APPROVED** [review]
Ok the migration script is triggered with upgrade test

File: app/setup_handlers.go (hunk 1 above)
Comment (kingpinXD) [reply app/setup_handlers.go:45]: We have always been using this same process 
We set the entire map to latest version 
	```
	for m, mb := range app.mm.Modules {
			vm[m] = mb.ConsensusVersion()
		}
```

**Pull request merged**